docs/using-the-python-driver/using-plugins/UsingTheLimitlessPlugin.md

```diff
@@ -0,0 +1,49 @@
+| `limitless_get_transaction_router_max_retries`          | Integer |    No    | This property is the max number of times the Limitless Connection Plugin will retry fetching available transaction routers and their load metrics. These retries will occur if the fetched transaction router information is `None` or empty. If this max is reached, a `SQLException` will be thrown. In this scenario, it is likely that the database is in an unhealthy state, and the thrown `SQLException` should be caught and handled by your application. <br><br>If your application prioritizes failing fast, then consider a lower value for this property. However, if your application prioritizes durability, then consider a higher value.                                                                                                                                                                                                                                                                                                                                         | `5`           | `10`        |
+| `limitless_get_transaction_router_retry_interval_ms`    | Integer |    No    | This property is the interval in milliseconds between retries of fetching available transaction routers and their load metrics. <br><br>If your application prioritizes failing fast, then consider a lower value for this property. However, if your application prioritizes durability, then consider a higher value.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | `300`         | `1000`      |
+
+### Use with other plugins
+The Limitless Plugin is compatible with authentication type plugins such as the IAM and AWS Secrets Manager Plugins.
+
+> [!IMPORTANT]\
+> The Failover, Host Monitoring, and Read Write Splitting Plugins are also compatible with the Limitless Plugin.  
+However, we don't recommend using them with the Limitless Plugin because they're not designed to be used with Aurora LimitlessDatabase. 
```

Comment:
```suggestion
However, we don't recommend using them with the Limitless Plugin because they're not designed to be used with Aurora Limitless Database. 
```